src/main/java/io/codelex/collections/practice/Exercise5.java

```diff
@@ -0,0 +1,24 @@
+package io.codelex.collections.practice;
+
+import java.util.Scanner;
+
+public class Exercise5 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter your number: ");
+        String number = sc.nextLine();
+        int sum = 0;
+        for (int i = 0; i < number.length(); i++) {
```

Comment:
Šis algoritms īsti ne līdz galam strādā.
"An integer is "happy" if repeatedly summing the squares of its digits eventually leads to the number 1."
Āķis ir tajā, ka ir atkārtoti jāsummē tie kāpinājumi. Jebšu, ja ir divciparu skaitlis 15, tad sanāk 1^2 + 5 ^ 2 = 1 + 25 = 26, un šis skaitlis 26 atkal ir jādala un jākāpina 2^2 + 6^2 = 4 + 36 = 40, atkal 4 ^ 2 + 0 ^ 2 = 16 + 0 = 16 u.t.t. bet Tavs algoritms tikai 1x iziet cauri tiem skaitļiem. Kā arī ir vēl āķis - ja to atkārto tik cik vajag, tad var ieiet bezgalīgā ciklā un tas jāpārtrauc, kad ir skaidrs, ka tas nav happy number. Un pārtraukt var tad, ja saskarās ar kādu iepriekš jau aprēķinātu skaitli.
Bet nu tā kā tas happy number aprēķins nav pats svarīgākais šajā, tad vari atstāt kā ir.